### PR TITLE
Set WAIT 2 for all ALTER TABLE REORGANIZE PARTITION calls

### PR DESCRIPTION
**Pull request by jcjones**

### CODE REVIEW

**Review by mcpherrinm — APPROVED** [review]
lgtm.  Per the mariadb manual, `wait 2` will wait to acquire locks for two seconds: https://mariadb.com/kb/en/wait-and-nowait/

Based on my reading of our metrics, this is an acceptable value.  Longer waits are quite rare.

Do we need to add any extra error handling to partman to handle the new timeout errors we'll get?

**Comment by jcjones** [comment]
We probably should, the MySQL command will return error, which is already accounted for, but the metrics wouldn't get updated. I've got an improvement, thanks @mcpherrinm!

File: partitionmanager/cli.py [hunk 1]
```diff
@@ -296,9 +296,15 @@ def do_partition(conf):
...
     )
+    metrics.describe(
+        "alter_errors",
+        help_text="Number of errors observed during ALTER commands",
+        type_name="counter",
+    )
 
     all_results = dict()
     for table in conf.tables:
+        time_start = None
```
Comment (mcpherrinm) [inline partitionmanager/cli.py:307]: I was confused when reviewing but I see we set time_start to now on line 345; this is just to pull it into a higher scope

**Review by mcpherrinm — APPROVED** [review]
Thanks!  The metric should be helpful.

**Pull request merged**